Rubberduck.RibbonDispatcher/Sample Ribbons/RibbonLibrary/src/RibbonCallbacksAdapter.bas

```diff
@@ -0,0 +1,226 @@
+Attribute VB_Name = "RibbonCallbacksAdapter"
+Option Explicit
+
+Public Const ModuleName   As String = "RibbonCallbacksAdapter."
+Public Const DefaultImage As String = "MacroSecurity"
+
+Public Enum ControlSize
```

Comment:
Why wrap the enum with the library early-bound by virtue of the `ViewModelFor` function signature right underneath it?

Reply:
The enum is ***public***; because it has to be for reason known only to COM and God.